examples/imitation_learning_openbot/utils.py

```diff
@@ -102,14 +103,14 @@ def generate_video(image_dir, video_path, rate, compress=False):
     process.wait()
 
 
-def plot_tracking_performance_spatial(poses_current, poses_desired, plot_path):
+def plot_tracking_performance_spatial(poses_tracked, poses_desired, plot_path):
```

Comment:
`poses_actual`

Reply:
Updated the names to `location` instead of `poses` as we are only passing in `locations`.